lib/awful/widget/watch.lua

```diff
@@ -60,22 +62,48 @@ local watch = { mt = {} }
 -- For "exit" reason it's the exit code.
 -- For "signal" reason — the signal causing process termination.
 --
--- @param[opt=wibox.widget.textbox()] base_widget Base widget.
+-- @tparam[opt=wibox.widget.textbox()] widget base_widget Base widget.
 --
 -- @return The widget used by this watch
-function watch.new(command, timeout, callback, base_widget)
-    timeout = timeout or 5
+function watch.new(command, interval, callbacks, base_widget)
```

Comment:
The actual parameter name and its documentation no longer matches: `callback` / `callbacks`.
Also, the whole table-stuff is undocumented...?